projects/tic_tac_toe/game.css

```diff
@@ -0,0 +1,40 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: x-large;
+}
+
+.flex-box-column-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.flex-box-row-container {
+  display: flex;
+}
+
+.grid-container {
+  display: grid;
+  background-color: black;
+  grid-template-columns: auto auto auto;
+  gap: 5px;
+  padding: 5px;
+}
+
+.grid-container > div {
+  text-align: center;
+  padding: 30px;
+  background-color: #eee;
+  height: 20px;
+  width: 20px;
+}
+
+.grid-container > div:hover {
```

Comment:
I recommend making the cell a slightly darker color when the user hovers over the box. Maybe `#ddd`. Up to you.